test/runnable/test19729.d

```diff
@@ -0,0 +1,61 @@
+//https://issues.dlang.org/show_bug.cgi?id=19729
+// PERMUTE_ARGS:
```

Comment:
Should have an empty `TEST_OUTPUT` to ensure no silent deprecations or other messages are triggered.

```
TEST_OUTPUT:
---
---
```

Or are we adding this by default now?

Reply:
Yes, empty `TEST_OUTPUT` is the default nowadays.